src/test/java/core/basesyntax/service/impl/DataParserServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParserService;
+import core.basesyntax.service.FileReadService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class DataParserServiceImplTest {
+    private static FileReadService fileReadService;
+    private static DataParserService dataParserService;
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+
+    @BeforeClass
+    public static void beforeClass() {
```

Comment:
```suggestion
    public static void init() {
```
fix in every test class